tailwind.config.js

```diff
@@ -18,7 +10,25 @@ const tailwindConfig = {
     },
     extend: {
       typography: {
-        DEFAULT: { css: disabledCss },
+        DEFAULT: {
+          css: {
+            'code::before': false,
+            'code::after': false,
+            'pre': false,
+            'code': false,
+            'pre code': false,
+            '*:not(pre) > code': {
```

Comment:
@tom-blake I think when everything is markdoc we can just add styling to a component.